apps/wallets/smart-wallet/react/src/app/utils/createAAWallet/createMetamaskWallet.ts

```diff
@@ -24,13 +23,9 @@ export const createMetamaskAAWallet = async (isProd: boolean) => {
                   clientApiKey: process.env.REACT_APP_CROSSMINT_API_KEY_STG || "",
               });
 
-        return await xm.getOrCreateWallet(
-            userIdentifier,
-            isProd ? SmartWalletChain.POLYGON : SmartWalletChain.POLYGON_AMOY,
-            {
-                signer: window.ethereum as any,
-            }
-        );
+        return await xm.getOrCreateWallet(userIdentifier, isProd ? Chain.POLYGON : Chain.POLYGON_AMOY, {
+            signer: window.ethereum as any,
+        });
```

Comment:
nit: might be a bit too much going on in one line
```suggestion
        const blockchain = isProd ? Chain.POLYGON : Chain.POLYGON_AMOY
        return await xm.getOrCreateWallet(userIdentifier, blockchain, { signer: window.ethereum as any });
```

Reply:
agree, but out of scope here. will cleanup demo later w/ jon